bot/code_review_bot/report/lando.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import structlog
+
+from code_review_bot import Level
+from code_review_bot.report.base import Reporter
+
+logger = structlog.get_logger(__name__)
+
+LANDO_MESSAGE = "The code review bot found {errors} errors which should be fixed to avoid backout and {warnings} warnings."
```

Comment:
Nit: we could use the singular form when there is a single error or a single warning